source/_integrations/comelit.markdown

```diff
@@ -37,6 +37,34 @@ There is support for the following platform types within Home Assistant:
 
```

Comment:
maybe outside the scope of this PR, but is there anything I need to have or have done before installing the integration? do I need to have set up the device, set a pin, have an app, a login and account or anything like that, for example?

Reply:
Yeah, the device needs to be setup; pin is optional, there are no login or account.